---
# Documentation: https://wowchemy.com/docs/managing-content/

title: IDS372, a phenotypically reconciled model for the metabolism of streptococcus
  pneumoniae strain R6
subtitle: ''
summary: ''
authors:
- O. Dias
- J. Saraiva
- C. Faria
- M. Ramirez
- F. Pinto
- I. Rocha
tags:
- Avirulent
- Genome-scale metabolic model
- IDS372
- Metabolic reconstruction
- Phenotypical reconciliation
- Streptococcus pneumoniae R6
categories: []
date: '2019-01-01'
lastmod: 2023-02-02T15:19:51Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2023-02-02T15:25:46.013295Z'
publication_types:
- '2'
abstract: Â© 2007 - 2019 Frontiers Media S.A. All Rights Reserved. A high-quality GSM
  model for Streptococcus pneumoniae R6 model strain (iDS372), comprising 372 genes
  and 529 reactions, was developed. The construction of this model involved performing
  a genome-wide reannotation to identify the metabolic capacity of the bacterium.
  A reaction representing the abstraction of the biomass composition was reconciled
  from several studies reported in the literature and previous models, and included
  in the model. The final model comprises two compartments and manifold automatically
  generated gene rules. The validation was performed with experimental data from recent
  studies, regarding the usability of carbon sources, the effect of the presence of
  oxygen, and the requirement of amino acids for growth. This model can be used to
  better understand the metabolism of this major pathogen, provide clues regarding
  new drug targets, and eventually design strategies for fighting infections by these
  bacteria.
publication: '*Frontiers in Microbiology*'
doi: 10.3389/fmicb.2019.01283
---
